src/manage.c

```diff
@@ -326,6 +336,7 @@ static void manage_acq()
   u8 prn = choose_prn();
   if (prn == (u8)-1)
     return;
+  gnss_signal_t sid = {.sat = prn};
```

Comment:
I couldn't be bothered to fix this now.  In the next SBAS PR, `choose_prn` will become `choose_sid` and will return the sid with constellation and band filled in.